web/static/web/css/header.css

```diff
@@ -1,144 +1,274 @@
+
+    /* nav */
+    --nav-space-between: 2em;
+    --bubble-background-extra-width: calc(1.5 * var(--nav-space-between));
+    --dur: 0.15s;
+    --font-size: 1.25em;
+
     display: grid;
-    grid-template-columns: minmax(120px, 1fr) 3fr;
-    grid-template-rows: 120px;
+    grid-template-columns: 1fr auto 1fr;
     width: 100%;
-    height: 120px;
-    background-color: rgb(33, 43, 52);
+    height: var(--header-height);
+    background-color: rgb(34, 43, 52);
 }
 
-.nav .menu {
-    align-self: center;
-    margin: 0;
-    padding: 0 30px;
+#header .item > .text {
+    color: #DCDDDE;
+}
+
+#header .item:hover > .text {
+    color: #FFF;
+}
+
+#header div.logo-centering {
+    display: none;
 }
 
-.nav .logo {
+#header .logo {
+    display: grid;
+    grid-template-columns: var(--logo-padding) var(--logo-space) minmax(var(--nav-space-between), var(--logo-padding));
     align-self: center;
-    padding: 0 30px;
 }
 
-.nav .logo img {
+#header .logo img {
+    display: none; /* Logo size changes at the bottom of this stylesheet */
+    margin: auto;
+    max-height: 50px;
     object-fit: contain;
 }
 
-.limg {
+#burger {
     display: none;
+    align-self: center;
+    justify-self: end;
+    margin-right: var(--burger-margin);
 }
 
-.limg.large {
-    display: inline;
+#burger:hover {
+    cursor: pointer;
 }
 
-.burger {
-    display: none;
-    margin-bottom: -10px;
+#nav {
+    font-size: var(--font-size);
 }
 
-.burger:hover {
-    cursor: pointer;
+#nav > .item,
+#side-nav > .item {
+    height: calc(var(--font-size) + 2em);
+    margin: calc(var(--nav-space-between) / 2);
+    padding: 0;
+    justify-content: center;
 }
 
-.burger {
-    align-self: center;
-    justify-self: center;
+#nav > .item:hover,
+#side-nav > .item:hover,
+#nav-user-dropdown.active {
+    /* Disables property from Semantic UI */
+    background-color: transparent;
 }
 
-.burger i {
-    color: #FFFFFF;
-    font-size: 30px;
+#nav > .item .bubble-background,
+#side-nav > .item .bubble-background {
+    position: absolute;
+    width: 100%;
+    height: var(--font-size);
+    border-radius: 2em;
+    opacity: 0;
+    transition: width var(--dur) linear, height var(--dur) linear, opacity var(--dur) ease-out;
 }
 
-.nav .ui.inverted.menu .item {
-    color: rgba(255, 255, 255, .7) !important;
+#nav > .item:hover .bubble-background,
+#side-nav > .item:hover .bubble-background {
+    width: calc(100% + var(--bubble-background-extra-width));
+    height: calc(var(--font-size) + 2em);
+    opacity: 1;
 }
 
-.lang-form {
-    display: none;
+#nav > .item > .text,
+#side-nav > .item > .text {
+    z-index: 1;
+    transition: color var(--dur) ease;
 }
 
-@media screen and (max-width: 900px) {
-    .limg.large {
-        display: none;
-    }
+#nav > .item .make_bg_yellow.bubble-background {
+    filter: brightness(90%);
+}
 
-    .limg.small {
-        display: none;
-    }
+#nav > .item .make_bg_turquoise.bubble-background {
+    filter: brightness(90%);
```

Comment:
Combine with the block above?

Reply:
I'd prefer not, because it was just a coincidence that both yellow and turquoise worked with the same brightness level; they should have independent properties.